ui/js/compareperf.js

```diff
@@ -119,6 +120,29 @@ perf.controller('CompareResultsCtrl', [
             newSignature: newSig
           }), function(kv) { return kv[0]+"="+kv[1] }).join("&");
 
+
+          // Construct the graph link for each row
+          var originalSeries = encodeURIComponent(JSON.stringify(
+                        { project: $scope.originalProject.name,
+                          signature: oldSig,
+                          visible: true}));
+
+          var newSeries = encodeURIComponent(JSON.stringify(
+                        { project: $scope.newProject.name,
+                          signature: newSig,
+                          visible: true}));
```

Comment:
A linter implies there could indentation improvements here but I left them the same as the ones in `CompareSubtestResultsCtrl`.